docs/supported-providers.md

```diff
@@ -130,12 +130,12 @@ Compose sample:
             - OPENVPN_PASSWORD=pass
 ```
 
-### If you only need to mount one file
+### Do not mount single config file
 
-You might not need to mount a folder of configs. You may just have one config file you want to use.
-In that case you can just mount it directly. Mounting it as `default.ovpn` will let you omit `OPENVPN_CONFIG` as well.
+Do not mount a single config directly. The container will fail if you try, since it causes sed errors when modify-openvpn-config.sh is executed.
+Instead mount the directory where the config exists.
```

Comment:
I think it is a good idea to add a comment like this but one can also disable the config modifications like I have using : MODIFY_CHOSEN_CONFIG and set it to false..